test/HardwareObjectsMockup.xml/mxcube-web/server.yaml

```diff
@@ -22,7 +24,7 @@ mxcube:
 
   usermanager:
     class: UserManager
-    inhouse_is_staff: True
+    inhouse_is_staff: true
```

Comment:
I don't know YAML in detail. Is`True` and `true`  equivalent in YAML? I.e. are the both treated as booleans? Or does this change the type from string to boolean?

If this is the latter case, when we need to check the code that consumes these values, so it still does the correct thing.

Reply:
Its treated the same as far as I know it results in a boolean

Reply:
This was triggering this rule: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy

Yes, YAML is a disaster :D

I checked the mxcube code that uses these values and it expects booleans, not strings, so I assume this change should be fine, seems like the tests still pass.

We could set the rule that all strings have to be in quotes, so that we can not get confused between strings and booleans (and other value types) any more, but I bet lots of people wold hate that rule :D

Reply:
Nice, then it's not a problem. Thanks for checking.